### PR TITLE
Closes #1411-[NTR]-fibroblast-like_synoviocyte

### LINKED ISSUE #1411: [NTR] Fibroblast-like synoviocyte
**Preferred term label:** Fibroblast-like synoviocyte

**Synonyms**: Synovial fibroblasts

**Definition**: Fibroblast-like synoviocytes are the resident stromal cells of the joints responsible for the production of immune-related cytokines and chemokines. These cells play a key role in the pathogenesis of rheumatoid arthritis and are essential for the maintenance of normal joint functions. Fibroblast-like synoviocytes from different joints have different properties, which may contribute to the characteristic pattern of joint involvement in different types of arthritis [PMID: 34433485; PMID: 34772990]

**Parent cell type term:** 
synovial cell (CL:0000214)
stromal cell (CL:0000499)

**What is the anatomical structure that the cell is a part of?**: synovial joint (UBERON:0002217)

**Your nano-attribution (ORCID)**: 0000-0002-0819-0473

**Any additional notes or concerns**: thanks

### PR DESCRIPTION
**Pull request by bvarner-ebi**
New term added: fibroblast-like synoviocyte

I also added a synonym (type B synoviocyte) discovered whilst reviewing the literature

### CODE REVIEW

**Comment by bvarner-ebi** [comment]
QC check is failing. Will have to investigate further.

```
2022-01-21 18:13:04,271 ERROR org.obolibrary.robot.ReasonOperation - Only equivalent classes that have been asserted are allowed. Inferred equivalencies are forbidden.
148
2022-01-21 18:13:04,272 ERROR org.obolibrary.robot.ReasonOperation - Equivalence: <http://purl.obolibrary.org/obo/CL_0000214> == <http://purl.obolibrary.org/obo/CL_4030001>
```

**Comment by shawntanzk** [comment]
Strange thing is I tried running ROBOT explain with ELK reasoner and it says "No explanations found."
Think I ran it correctly? 
```
sh run.sh robot explain --input cl-edit.owl --reasoner ELK --axiom "'synovial cell' EquivalentTo 'fibroblast-like synoviocyte'" --explanation tmp/explain.md
```

But manually looking at it:

Synovial cell equiv to: 
'meso-epithelial cell'
 and ('part of' some 'synovial membrane of synovial joint')
 
 fibroblast-like synoviocyte inferred subclass of 'meso-epithelial cell' 
 and 'synovial membrane of synovial joint' is part_of 'synovial joint' 

therefore they are equiv to each other which is not allowed

**Comment by dosumis** [comment]
Explanation not working in Protege either and I can't see why we're getting equivalence between these two terms by simple inspection.

**Comment by dosumis** [comment]
synovial cell / synoviocyte appears to be a term the groups two unrelated cell types:

Type A are macrophages, Type B are fibroblast-like synoviocytes
see https://en.wikipedia.org/wiki/Synovial_membrane#Synovial_cells
& https://pubmed.ncbi.nlm.nih.gov/10770586/

I think we have a couple of options 
1. Make a grouping term that manually classifies terms for type A and type B - both more rigorously defined.   The current term should probably be the grouping term.
2. Make separate terms for each type and rely on labels, synonyms and comments for indexing and disambiguation. 

Note -  it might be possible to us GO to give type B cells a partly functional logical definition to do with invovlement in secretion of synovial fluid (components).

**Comment by dosumis** [comment]
Note - GO has a term that reference CL synoviocyte.   The appear to use the term to refer to the fibroblast-like cell type.

**Comment by dosumis** [comment]
Should have looked more carefully at the terms we have:

image

Suggested fix

- [ ] remove newly added term (it's the same as type B)
- [ ] Make synovial cell a broad grouping term.  It needs a very broad grouping classification.  I think it probably needs to be part of synovial join rather than synovial membrane (check on the location of type A synoviocytes).  No EC def.  Remove/move all synoyms, xrefs and definition text that refer to type B cells.
- [ ]  type B could have the EC def currently on the new term
- [ ] consider adding subClassOf axioms to type B to record that they 'capable of' are 'hyaluronic acid' secretion. (You might need to use composition to specify this - use the patterns in go-plus to guide this).
- [ ] Inform GO that GO_0002941  should reference type B
- [ ] Look into adding an additional immune/macrophage classification & perhaps developmental origin to type A

**Comment by bvarner-ebi** [comment]
Thank you for the feedback, @shawntanzk and @dosumis. I will edit accordingly.

**Comment by bvarner-ebi** [comment]
Comments below:

> Suggested fix
> 
> * [x]  remove newly added term (it's the same as type B)

^ DONE

> * [ ]  Make synovial cell a broad grouping term.  It needs a very broad grouping classification.  I think it probably needs to be part of synovial join rather than synovial membrane (check on the location of type A synoviocytes).  No EC def.  Remove/move all synoyms, xrefs and definition text that refer to type B cells.

^ Broadened definition for 'synovial cell'. Unsure if I should remove the 'meso-epithelial' subclass.... Based on references, type A synoviocytes are located in the synovial membrane (the superficial layer of the synovial intimate). Would  ('part of' some 'synovial membrane of synovial joint') under subclass be appropriate? 

> * [x]   type B could have the EC def currently on the new term

^ Does EC def = Equivalence axiom? Definitions updated.

> * [ ]  consider adding subClassOf axioms to type B to record that they 'capable of' are 'hyaluronic acid' secretion. (You might need to use composition to specify this - use the patterns in go-plus to guide this).
 
^ I requested term for 'hyaluronic acid secretion' in GO. https://github.com/geneontology/go-ontology/issues/22715. Once created and imported to CL, I will add. 

> * [x]  Inform GO that GO_0002941  should reference type B

^ Requested. https://github.com/geneontology/go-ontology/issues/22713

> * [ ]  Look into adding an additional immune/macrophage classification & perhaps developmental origin to type A

^ I added subClassOf axiom 'derives from' some 'mononuclear cell' to type A. When I add 'macrophage' as a subclass and run reasoner, I get OWL:nothing inferred under EA, so I did not include that.

**Comment by shawntanzk** [comment]
> ^ I added subClassOf axiom 'derives from' some 'mononuclear cell' to type A. When I add 'macrophage' as a subclass and run reasoner, I get OWL:nothing inferred under EA, so I did not include that.

Could you press the "?" on the OWL:nothing inference and check the explanation for why please? 
(alternatively you can run `sh run.sh make explain_unsat` and you should have a explain_unsat.md in your tmp folder that you can put here directly)

**Comment by dosumis** [comment]
Was going to suggest typing as 'stromal cell' - you beat me to it!

**Comment by bvarner-ebi** [comment]
Additional comments below:
 

> * [x]  Make synovial cell a broad grouping term.  It needs a very broad grouping classification.  I think it probably needs to be part of synovial join rather than synovial membrane (check on the location of type A synoviocytes).  No EC def.  Remove/move all synoyms, xrefs and definition text that refer to type B cells.

^ added ('part of' some 'synovial membrane of synovial joint') to type A cells

> * [ ]  consider adding subClassOf axioms to type B to record that they 'capable of' are 'hyaluronic acid' secretion. (You might need to use composition to specify this - use the patterns in go-plus to guide this).

^ I added the following to type B cells:
‘capable of’ some ‘hyaluronan biosynthetic process'
and
produces some synovial fluid

Question: should definition for UBERON:0001090 'synovial fluid' be updated?
"Joint fluid is a transudate of plasma that is actively secreted by synovial cells." -->
Joint fluid is a transudate of plasma that is actively secreted by type B synovial cells.

> * [ ]  Look into adding an additional immune/macrophage classification & perhaps developmental origin to type A

^ I added 'capable of' some phagocytosis and, as noted above, 'derives from' some 'mononuclear cell'

**Comment by bvarner-ebi** [comment]
Hi, @shawntanzk, @paolaroncaglia, this PR is ready for review.
Let me know if there are other editors who could also review- I don't want to inundate you with all my review requests.

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -5135,18 +5134,14 @@ SubClassOf(obo:CL_0000213 obo:CL_0000215)
...
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000214 "CALOHA:TS-0995"^^xsd:string)
-AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000214 "FMA:66786"^^xsd:string)
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000214 "synoviocyte"^^xsd:string)
-AnnotationAssertion(oboInOwl:hasNarrowSynonym obo:CL_0000214 "hyaluronic acid secreting cell"^^xsd:string)
 AnnotationAssertion(oboInOwl:hasOBONamespace obo:CL_0000214 "cell"^^xsd:string)
 AnnotationAssertion(rdfs:label obo:CL_0000214 "synovial cell"^^xsd:string)
-EquivalentClasses(obo:CL_0000214 ObjectIntersectionOf(obo:CL_0002078 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002018)))
-SubClassOf(obo:CL_0000214 obo:CL_0000153)
 SubClassOf(obo:CL_0000214 obo:CL_0000213)
 SubClassOf(Annotation(oboInOwl:is_inferred "true") obo:CL_0000214 obo:CL_0002078)
```
[the remark this answers is not in the document]
Comment (bvarner-ebi) [reply src/ontology/cl-edit.owl:5143]: Thanks, @shawntanzk.
I think this is OK.
For example, 
for type A: they are derived from monocytes, which come from a myeloid precursor, which is derived from lateral plate **mesoderm**/hemangioblast (https://en.wikipedia.org/wiki/List_of_human_cell_types_derived_from_the_germ_layers#Lateral_plate_mesoderm/hemangioblast).

For type B: PMID:15634908: "Fibroblast-like synoviocytes of **mesenchymal origin** express functional B cell-activating factor..."

**Pull request merged**